extensions/amp-story/1.0/amp-story.js

```diff
@@ -1063,6 +1065,23 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Parse doc for polls and quizzes, and update the story-state.
+   */
+  initializeInteractiveComponents_() {
+    toArray(
+      this.element.querySelectorAll(
+        'amp-story-interactive-quiz, amp-story-interactive-poll'
+      )
+    ).forEach((element) => {
+      whenUpgradedToCustomElement(element).then((el) =>
+        el.getImpl().then((impl) => {
+          impl.updateStoryStoreState_();
+        })
+      );
+    });
+  }
```

Comment:
When the story gets created, all the interactive components will update the state so that the state knows how many interactives there are. This is necessary to know when the results are finished, and in the future to know what percentage of answers are completed/correct